community/releases/next-release-preview/index.md

```diff
@@ -55,3 +55,12 @@ Spinnaker's UI has changed! An application's nested menus are now represented as
 This change should not introduce any interruptions to a vanilla install of `deck`. However, if you've already made navigational changes to your group's instance of `deck` or created custom banners/headers for your app, you may need to make updates. The pattern for creating new routes in the side nav can be observed in the feature's PR:
 
 https://github.com/spinnaker/deck/pull/8239
+
+### (Breaking Change) Spinnaker Dockerfile GID/UID changes
```

Comment:
I'd be inclined to put the breaking changes a the top, what do you think?

Reply:
+1

Reply:
good call - done!